examples/nginx.conf

```diff
@@ -25,7 +25,7 @@ events {
     worker_connections 1024;
 }
 http {
-    lua_package_path "/Users/wusheng/Documents/GitHub/skywalking-nginx-lua/lib/?.lua;;";
+    lua_package_path "/Users/hua/workspace/skywalking/skywalking-nginx-lua/lib/?.lua;;";
```

Comment:
I think it is better to revert this? At least, we should not change this from one developer's path to another's.